jobs/metricsforwarder/spec

```diff
@@ -0,0 +1,105 @@
+  hooks/post-start.sh.erb: bin/hooks/post-start.sh
+  hooks/post-stop.sh.erb: bin/hooks/post-stop.sh
+  
+packages:
+  - java
+  - db
+  - metricsforwarder
+  - common
+
+properties:
+  autoscaler.metricsforwarder.logging.level:
+    description: "The level of logging"
+    default: "info"
+  autoscaler.metricsforwarder.server.port:
+    description: "Port on which the metricsforwarder server will listen"
+    default: 6201
+  autoscaler.metricsforwarder.loggregator.metron_address:
+    description: "IP address and port where the metron agent is running"
+    default: "127.0.0.1:3458"
+
+  
+  autoscaler.metricsforwarder.loggregator.tls.cert:
+    description: "PEM-encoded tls client certificate to connect to loggregator ingress client"
+  autoscaler.metricsforwarder.loggregator.tls.key:
+    description: "PEM-encoded tls client key to connect to loggregator ingress client"
+  autoscaler.metricsforwarder.loggregator.tls.ca_cert:
+    description: "PEM-encoded ca certificate of loggregator ingress client"
+
+  autoscaler.policy_db.address:
+    description: "IP address on which the policydb server will listen"
+    default: "autoscalerpostgres.service.cf.internal"
+  autoscaler.policy_db.databases:
+    description: "The list of databases used in policydb database including name"
+  autoscaler.policy_db.db_scheme:
+    description: "Database scheme to be used to access policydb"
+  autoscaler.policy_db.port:
+    description: "Port on which the policydb server will listen"
+  autoscaler.policy_db.roles:
+    description: "The list of database roles used in policydb database including name/password"
+  autoscaler.policy_db.tls.ca:
+    default: ''
+    description: 'PEM-encoded ca certificate of tls database server'
+  autoscaler.policy_db.sslmode:
+    default: disable
+    description: "The sslmode to connect to postgres server"
+
+  autoscaler.policy_db_connection_config.max_open_connections:
+    default: 100
+  autoscaler.policy_db_connection_config.max_idle_connections:
+    default: 10
+  autoscaler.policy_db_connection_config.connection_max_lifetime:
+    default: 60s
+
+  autoscaler.metricsforwarder.require_consul:
+    description: "Whether to start consul agent"
+    default: false
+
+  autoscaler.metricsforwarder.cache_ttl:
+    description: "The time interval to mark cache entries as expired in metric forwarder"
+    default: 900s
+  autoscaler.metricsforwarder.cache_cleanup_interval:
+    description: "The time interval to purge expired cache entries in metric forwarder"
+    default: 6h
+  autoscaler.metricsforwarder.policy_poller_interval:
+    description: "The time interval to refresh cached application information from policy database"
+    default: 60s
+
+  
+  autoscaler.metricsforwarder.health.port:
+    description: "The listening port of health endpoint"
+    default: 6403
+
+  autoscaler.metricsforwarder.hooks.pre_start:
+    description: "Script to run before starting metricsforwarder"
+    default: ''
+    example: |
```

Comment:
why we need example?

Reply:
@boyang9527  it is a sample to show how to add hooks.